### PR TITLE
Make URLs with spaces pass the onboarding.

### LINKED ISSUE #127: Repos with spaces in the URL should be accepted
https://github.com/italia/developers-italia-onboarding/blob/master/src/validator.js#L22

### PR DESCRIPTION
**Pull request by bfabio**
Fix #127.

### CODE REVIEW

File: test/validator.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    expect(url('http://subdomain.example.com/')).to.equal(VALIDATION_OK);
+});
+
+lab.test('non-HTTP URLs', () => {
+    expect(url('ftp://example.com/')).to.equal(VALIDATION_INVALID_URL);
+    expect(url('foobar://example.com/')).to.equal(VALIDATION_INVALID_URL);
+});
+
+lab.test('spaces in URL', () => {
+    expect(url('https://example.com/spaces in url')).to.equal(VALIDATION_OK);
```
Comment (libremente) [inline test/validator.js:24]: Should we add the `%20` case?

Comment (bfabio) [reply test/validator.js:24]: Sure, done.

**Comment by bfabio** [comment]
I'm wondering why the CI is not triggering.

File: src/validator.js [hunk 2]
```diff
@@ -12,23 +12,26 @@ const adapter = new FileSync(whitelistFile);
...
-  return VALIDATION_OK;
 }
 
 /**
  * Controlla il formato del numero telefonico
- * 
+ *
  * @param {string} phone numero telefonico da verificare
  * @return Un oggetto di tipo ValidatorResult
  */
```
Comment (sebbalex) [inline src/validator.js:37]: I'd translate also these comments lines.

Comment (bfabio) [reply src/validator.js:37]: @sebbalex I'd do that in another PR, wdyt?

Comment (sebbalex) [reply src/validator.js:37]: :+1:

**Review by sebbalex — APPROVED** [review]
lgtm

**Review by libremente — APPROVED** [review]

**Pull request merged**